src/components/shared/TreeView.js

```diff
@@ -490,14 +512,14 @@ class TreeView<
   }
 
   _onKeyDown(event: KeyboardEvent) {
-    if (event.ctrlKey || event.altKey || event.metaKey) {
-      return;
-    }
+    const hasModifier = event.ctrlKey || event.altKey || event.metaKey;
+    const isArrowKey = event.key.startsWith('Arrow');
+    const isAsteriskKey = event.key === '*';
+    const isEnterKey = event.key === 'Enter';
 
-    if (event.keyCode < 37 || event.keyCode > 40) {
-      if (event.keyCode !== 0 || String.fromCharCode(event.charCode) !== '*') {
-        return;
-      }
+    if (hasModifier || (!isArrowKey && !isAsteriskKey && !isEnterKey)) {
```

Comment:
(not for this patch) I consider it's surprising that "alt + right arrow" doesn't expand all, because "alt + click" does.

Reply:
Ah, that makes sense to me.